rdmo/projects/views/project.py

```diff
@@ -126,14 +126,16 @@ def get_context_data(self, **kwargs):
                                                  .filter_catalog(self.object.catalog) \
                                                  .filter_group(self.request.user) \
                                                  .filter_availability(self.request.user).exists()
+        context['ancestors_import'] = ancestors.filter_user(user=self.request.user) \
```

Comment:
I've changed the query to `filter_user` instead of `filter`.
I thought that sometimes the projects that should be in the import were filtered out by `filter`?

Reply:
`filter_user` takes `admins` and `site_managers` into account, so this makes sense here.

Reply:
On the second thought, I don't think this aproach is correct, since it will not account for "role inheritance". If you are the owner of a parent project, you are automatically owner of all descendants. I think it would be better to use the `rules` in the templates to filter the projects.

Reply:
Alternatively we can use something like:

```
        queryset = Project.objects.filter(user=self.request.user)
        for instance in queryset:
            queryset |= instance.get_descendants()
        queryset = queryset.distinct()
```

which is the query for `/projects/`.